app/test/frontend/golden/pkg_show_page_flutter_plugin.html

```diff
@@ -257,7 +257,7 @@ <h3 class="title">More</h3>
             </p>
           </aside>
         </div>
-        <script type="application/ld+json">{"@context":"http:\u002f\u002fschema.org","@type":"SoftwareSourceCode","name":"flutter_titanium","version":"1.10.0","description":"flutter_titanium - flutter_titanium is awesome","url":"https:\u002f\u002fpub.dev\u002fpackages\u002fflutter_titanium","dateCreated":"%%published-timestamp%%","dateModified":"%%updated-timestamp%%","programmingLanguage":"Dart","image":"https:\u002f\u002fpub.dev\u002fstatic\u002fimg\u002fpub-dev-icon-cover-image.png"}</script>
+        <script type="application/ld+json">{"@context":"http:\u002f\u002fschema.org","@type":"SoftwareSourceCode","name":"flutter\u005ftitanium","version":"1.10.0","description":"flutter\u005ftitanium - flutter\u005ftitanium is awesome","url":"https:\u002f\u002fpub.dev\u002fpackages\u002fflutter\u005ftitanium","dateCreated":"%%published-timestamp%%","dateModified":"%%updated-timestamp%%","programmingLanguage":"Dart","image":"https:\u002f\u002fpub.dev\u002fstatic\u002fimg\u002fpub-dev-icon-cover-image.png"}</script>
```

Comment:
`/` was always escaped because it can be part of a closing tag. `https://` could be a special case, but if the search engine do proper parsing, it shouldn't really matter.